app/scripts/views/elements/README.md

```diff
@@ -0,0 +1,14 @@
+# Custom Elements
```

Comment:
😍 directory-specific readme files! Great idea, it never occurred to me to do this before!